src/assets/scss/_ads.scss

```diff
@@ -8,28 +8,27 @@
   position: static;
   display: block;
   max-width: 400px;
-  padding: 12px 12px 12px 154px;
-  margin: 1.5rem auto 0;
+  padding: 15px 15px 15px 160px;
+  margin: 2rem 0;
```

Comment:
Maybe the surrounding HTML code is different in the blog, but `margin: 2rem 0;` causes a regression making the ad not horizontally centered at some breakpoints:

<img width="956" alt="Screenshot 2024-04-03 at 09 04 52" src="https://github.com/twbs/blog/assets/17381666/b3579545-99a5-4ef4-aeb4-1551dc6d4f1a">

The easiest fix would be to do `margin: 2rem auto 0;` instead.

If we want exactly the same Scss file, we should rather dig into the HTML structure surrounding the ad "component".

Reply:
Feel free to push your changes as always

Reply:
What's your preferred approach? Having exactly the same Sass file for an easier maintenance?

Reply:
I took time to dig into this topic in detail and pushed https://github.com/twbs/blog/pull/443/commits/5febd2894d277203d3763e32f2105d638a36dbaf.

We've gathered upstream `_ad.scss` as is. However, upstream, there's an extra rule for `#carbonads` in `site/assets/scss/_masthead.scss`:

```scss
#carbonads { // stylelint-disable-line selector-max-id
  margin-inline: auto;
}
```

Ads are displayed in different contexts upstream so that an extra rule is not always needed. For the blog, it must be used. So I've added the extra `auto` directive and added a comment to try to show that this is specific.
You can remove or reword this comment the way you think it's going to be the most helpful in terms of maintenance; our future selves should understand at some point the following:
* blog: `_ad.scss` = upstream: `_ad.scss` + extra `auto` rule.